qa/workunits/rbd/rbd_mirror_snapshot.sh

```diff
@@ -440,6 +440,16 @@ wait_for_image_replay_started ${CLUSTER1} ${POOL} ${image}
 wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+replaying'
 compare_images ${POOL} ${image}
 
+testlog "TEST: request image resync while no known primary exists"
+demote_image ${CLUSTER1} ${POOL} ${image}
+request_resync_image ${CLUSTER1} ${POOL} ${image} image_id
+wait_for_image_present ${CLUSTER1} ${POOL} ${image} 'present' ${image_id}
+wait_for_image_present ${CLUSTER1} ${POOL} ${image} 'present'
+wait_for_image_replay_started ${CLUSTER1} ${POOL} ${image}
+wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+stopped'
```

Comment:
nit: you could add your new status message in this test as well
`wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+stopped' 'resync invalid, remote image is non-primary'`